src/interfaces/IFFI.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.27;
+
+/// @dev Interface for foreign function interface (FFI) contracts
+interface IFFI {
```

Comment:
Curious why this isn't specific to Across, e.g. `IFFIAcross`, since different contracts will have different addresses, right?

Reply:
My original intention was to have a single interface file where all the FFIs are defined. But now that we have `FFI.sol`, there's not much benefit to keep all the FFIs in one interface. I updated this to be `IAcrossFFI.sol`